src/components/Header/navbar.tsx

```diff
@@ -53,8 +62,4 @@ export const NavBar = ({match, isAuthenticated}: any) => {
   );
 };
 
-export default withRouter(
```

Comment:
why do you remove it?

1) now you've subscribed to the whole rootStore which i will plan to change in the near future
2) now i understand why you need `any` in props but this is not correct as well
3) component API becomes worse